uint32 motorLeftEncoderCount
uint32 motorRightEncoderCount
